src/cfml/system/BaseCommand.cfc

```diff
@@ -252,10 +252,11 @@ component accessors="true" singleton {
 	/**
 	 * This will open a URL in the user's browser
  	 **/
-	function openURL( theURL ) {
+	function openURL( theURL, browser = "" ) {
 		// Defer to "browse" command.
 		command( "browse" )
 			.params( arguments.theURL )
+			.params( arguments.browser )
```

Comment:
While this is valid, I usually only call the `.params()` method once if I know all the params I'm sending.  You can just do this:
```js
.params( arguments.theURL, arguments.browser )
```
Also, I realized you may be wondering why the variable `theURL` has the word "the" in it when I asked you to remove it from the "browser" parameter .  The reason is what `url` is a reserved variable name in CFML for the url scope and so I never create a variable of my own called `url` as it would create conflicts.  That's why this argument was called `theURL`, but that doesn't apply to `browser`.